packages/xdl/src/ProjectSettings.ts

```diff
@@ -165,6 +237,10 @@ The "packager-info.json" file contains port numbers and process PIDs that are us
 
 The "settings.json" file contains the server configuration that is used to serve the application manifest.
 
+> What is \`devices.json\`?
```

Comment:
```suggestion
> What does the "devices.json" file contain?
```

for consistency with the other headings

Reply:
@EvanBacon [suggested this change](https://github.com/expo/expo-cli/pull/4087#discussion_r779807142). Maybe we should make all the headers consistent with this new one?